server/README.md

```diff
@@ -73,3 +73,16 @@ make migration
 ```bash
 bash ./scripts/create-migration.sh
 ```
+
+## Working with test-data
+Initially no test data are loaded into the memory. If you require sample data provided by the app you can set a 
+flag to your environment `LOAD_TEST_DATA` or change the boolean flag [here](vars.py). Additionally, the test-data can
+be modified with a patient containing a time-limited state. The timelimit of the state can also be configured [here](vars.py).
+Loading test data creates two execution in the memory containing the same content despite the players' id. The details 
```

Comment:
[...] two execution**s** ~~the~~ in memory [...] content ~~despite~~ **except** the player **IDs**